main.py

```diff
@@ -4,13 +4,22 @@
 import urllib.request as request
 import sys
 
+regex_doi = r"doi\s?=\s?[{\"']([^}\"']+)[}\"']"
```

Comment:
Doesn't this also capure `doi = "10.1109/tvcg.2018.2865240",`? Is it valid bib?

Reply:
Yes it does. I believe it's valid bib. At least https://biblatex-linter.herokuapp.com/ doesn't complain and neither does my compiler.

Reply:
Cool, good to know.